### PR TITLE
Vchuprakov

### PR DESCRIPTION
**Pull request by victorxup**
tasks1-2-3-done

### CODE REVIEW

**Comment by victorxup** [comment]
tasks1-2-3-done
and some comments deleted

**Comment by victorxup** [comment]
.

File: src/main/java/com/epam/university/java/core/task005/PiHolderImpl.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+     * numerator.
+     */
+    private int first;
+
+    /**
+     * denominator.
+     */
+    private int second;
+
+    PiHolderImpl(final int first, final int second) {
```
Comment (aabarmin) [inline src/main/java/com/epam/university/java/core/task005/PiHolderImpl.java:18]: Constructor should be public

Comment (victorxup) [reply src/main/java/com/epam/university/java/core/task005/PiHolderImpl.java:18]: why?
we are not using this class outside of the package.
by the way, IDEA tip tells me about it, and highlights the modificator.
that's annoying

Comment (aabarmin) [reply src/main/java/com/epam/university/java/core/task005/PiHolderImpl.java:18]: Okay

**Pull request merged**